api/envoy/api/v2/core/grpc_service.proto

```diff
@@ -86,6 +86,46 @@ message GrpcService {
         }
       }
 
+      // Security token service configuration that allows Google gRPC to
+      // fetch security token from an OAuth 2.0 authorization server.
+      // See https://tools.ietf.org/html/draft-ietf-oauth-token-exchange-16 and
+      // https://github.com/grpc/grpc/pull/19587.
```

Comment:
Seems this feature is marked experimental in gRPC, how do we want to handle this for API stability? @htuch WDYT? should we create a v2alpha for this one?

Reply:
@jboeuf is there any timeline to make this a stable feature?

Reply:
There's no timeline that I'm aware of just yet. I think that we would like first to make sure that the feature is going to work for your use case. Once this is confirmed, we'll probably need a gRPC RFC for it graduate.

@jiangtaoli2016 FYI.

Reply:
I think it's fine to use it as is; folks using Google gRPC needs to be aware of the caveats of the external dep.